docs/Getting Started Part 2/index.md

```diff
@@ -0,0 +1,323 @@
+# Getting Started Part 2
+
+## Connecting HMI to SDL
+
+WebSocket is the primary means of communicating with the SDL Core component from the vehicle. In a basic example, an HTML5 HMI would use a native WebSocket library to communicate with SDL Core.
+
+The HMI Adapter must:
+
+!!! must
+  * be installed on the same vehicle HU OS where SDL is installed, or the HMI must be able to be networked to SDL and address it via a static IP address.
+  * create and initialize components which are defined in the HMI_API specification for the version of SDL which is running on the vehicle HU. (BasicCommunication, UI, Buttons, VR, TTS, Navigation, VehicleInfo, RC, AppService)
```

Comment:
Should we list interfaces here? i ask because it's harder to maintain and might be confusing for devs later

Reply:
_Ill add "for example" to preface_